open-company/open-company-web
# Growth section

card: https://trello.com/c/NCExXzwN

To test:
- use the data imported as the trello card suggests
- change and save the growth and the finances notes
- check all the metrics of all the companies

NB: this also fixes the first 2 elements of the checklist here: https://trello.com/c/NlXTc1vJ

There may be 2 problems:
1) if i PATCH a company with a body of: `{:notes {:body "Body"}}` i get back the section with the same author and updated-at in the section and in the notes
2) importing Thieve Hive i find some metrics that have a target but not a value (check [here](https://github.com/belucid/open-company-infrastructure/blob/master/samples/thrive-hive.edn#L799-L801))
